src/nbl/builtin/template/CArchive.h.in

```diff
@@ -0,0 +1,41 @@
+#ifndef _@_GUARD_SUFFIX_@_C_ARCHIVE_H_
+#define _@_GUARD_SUFFIX_@_C_ARCHIVE_H_
+
+#include <nbl/system/CFileArchive.h>
+#include "@NBL_BS_HEADER_FILENAME@"
+
+namespace @_NAMESPACE_@
+{
+	constexpr std::string_view pathPrefix = "@_PATH_PREFIX_@";
+	constexpr bool hasPathPrefix(std::string_view str) { return str.find(pathPrefix) == 0ull; }
+	
+	std::pair<const uint8_t*, size_t> get_resource_runtime(const std::string& filename);
+
+	class CArchive final : public nbl::system::CFileArchive
+	{
+		public:
+			inline CArchive(nbl::system::path _path, nbl::system::logger_opt_smart_ptr&& logger)
+				: nbl::system::CFileArchive(std::move(_path),std::move(logger), getItemList()) {}
```

Comment:
Path should be hardcoded, its the `pathPrefix`

Reply:
corrected in following commit